XCUITests/FxScreenGraph.swift

```diff
@@ -186,3 +186,14 @@ func createScreenGraph(app: XCUIApplication) -> ScreenGraph {
 
     return map
 }
+
+extension Navigator {
+    func openNewURL(urlString: String) {
+        let newTabScreen = map.namedScenes[NewTabScreen]!
+        let app = map.app
+        newTabScreen.addEdge(BrowserTab) { _ in
+            app.textFields["url"].tap()
```

Comment:
This second approach seems the simplest/most elegant – you don't need to make any changes to `ScreenGraph` at all then. I'd keep a route open the `BrowserTab` by having a default URL, just so other tests can get to it without thinking about it.